WeatherForecast/IsRoleDemo/IsRoleDemo.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
upgrade version to 6

Reply:
net 6.0 does not support IdentityServer4.AspNetIdentity, may stay with 3.1 because identity server 4 no more open source for 6.0 or take some advice